lnbits/core/static/js/wallet.js

```diff
@@ -668,7 +668,14 @@ new Vue({
       })
     },
     exportCSV: function () {
-      LNbits.utils.exportCSV(this.paymentsTable.columns, this.payments)
+      let columns = this.paymentsTable.columns
```

Comment:
Please add a short comment here why you're doing this, nobody will understand when looking at it in two years.